docs/reference/api/platform/Debug.md

```diff
@@ -1,11 +1,36 @@
 ## Debug
 
-[Add description here.]
+Mbed OS provides a set of debug functions that you can use to output debug messages to `STDIO` at runtime. `mbed_debug.h` declares these functions, which are available only in debug builds.
 
-### Debug class reference
+### Debug functions reference
 
-[Embed class reference here. If you've never done this before, please ask your editor.]
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__debug.html)
```

Comment:
Query: Does this link work for you? I can't get it to work.

Reply:
Actually it doesn't work now, it depends on my other change coming into mbed-os side.